apollo-runtime/src/main/java/com/apollographql/android/ApolloCall.java

```diff
@@ -10,12 +10,18 @@
 
 public interface ApolloCall {
 
-  void cancel();
-
   @Nonnull <T extends Operation.Data> Response<T> execute() throws IOException;
 
   @Nonnull <T extends Operation.Data> ApolloCall enqueue(@Nullable Callback<T> callback);
 
+  @Nonnull ApolloCall network();
```

Comment:
I'm not sure if there is any gain from it but another way to set this up is that each of these methods return a subclass such as `NetworkCall`,`CacheCall` `NetworkBeforeStaleCall` this way its a more polymorphic solution of instanceOf checks rather than checking flags.